system-contracts/bootloader/bootloader.yul

```diff
@@ -399,12 +399,12 @@ object "Bootloader" {
                 ret := add(TX_DESCRIPTION_BEGIN_BYTE(), mul(MAX_TRANSACTIONS_IN_BATCH(), TX_DESCRIPTION_SIZE()))
             }
 
-            /// @dev The memory page consists of 30000000 / 32 VM words.
+            /// @dev The memory page consists of 54000000 / 32 VM words.
             /// Each execution result is a single boolean, but
             /// for the sake of simplicity we will spend 32 bytes on each
             /// of those for now.
             function MAX_MEM_SIZE() -> ret {
-                ret := 30000000
+                ret := 54000000
```

Comment:
From the comments from `system-contracts/scripts/preprocess-bootloader.ts`, the memory for l1 messages has at least by `(1327000 - 451000) * 32` bytes, i.e. 28mln, so i would suggest to recalculate this value (or fix the comment)